### PR TITLE
7.0 update travis

### PR DESCRIPTION
**Pull request by yvaucher**
Superseeds #13

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Can you please add Transifex?

**Comment by yvaucher** [comment]
@pedrobaeza Fixed

File: README.md [hunk 1]
```diff
@@ -17,3 +17,7 @@ Please don't hesitate to suggest one of your module to this project. Also, you m
  - https://github.com/OCA/stock-logistics-tracking
  - https://github.com/OCA/stock-logistics-workflow
  - https://github.com/OCA/stock-logistics-warehouse
+
+Translation Status
+------------------
+[![Transifex Status](https://www.transifex.com/projects/p/OCA-stock-logistics-barcode-8-0/chart/image_png)](https://www.transifex.com/projects/p/OCA-stock-logistics-barcode-8-0)
```
Comment (pedrobaeza) [inline README.md]: s/8-0/7-0

**Comment by pedrobaeza** [comment]
Please squash your commits

**Comment by yvaucher** [comment]
@pedrobaeza first commit is @gurneyalex commit and the 2 other are 2 different changes. So my vote goes to keep this PR as it is. I'll fix the version error thought.

**Comment by yvaucher** [comment]
Fixed

**Comment by pedrobaeza** [comment]
OK

:+1:

**Comment by lmignon** [comment]
:+1:

File: .travis.yml [hunk 2]
```diff
@@ -13,22 +13,27 @@ python:
   - "2.7"
 
 env:
-  - VERSION="7.0" ODOO_REPO="odoo/odoo"
-  - VERSION="7.0" ODOO_REPO="OCA/OCB"
+  global:
+  - VERSION="7.0" ODOO_REPO="odoo/odoo" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
+  - TRANSIFEX_USER='transbot@odoo-community.org'
+  - secure: vSumRFP7qtgd8qERNnGwxxlb8hxPUyks3tUJ3EKOUE88PKEb/0ofvfjtyNa8ZHN+OMl7exod3Nuh1ejYw6uMfw8WQYjjMyPJL7sf8OK36CMU0drlj1m4/ohWn4BRyDeHL2GAymDG/TEHMNgY2s1Y2ASC9vhXOyzaj95VQhZH8ak=
```
Comment (tafaRU) [inline .travis.yml]: This seems wrong, it refers to stock-logistics-transport and not to stock-logistics-barcode. See https://github.com/OCA/maintainer-quality-tools/issues/194#issue-79502151

Comment (yvaucher) [inline .travis.yml]: You are damn right mixed my PR between the 2 repo

**Comment by yvaucher** [comment]
@tafaRU Thanks fixed. And fixed it too on #19

**Comment by tafaRU** [comment]
@yvaucher here the token is still wrong.

**Comment by yvaucher** [comment]
@tafaRU Indeed it was. :eyes: 

fixed

**Comment by tafaRU** [comment]
@yvaucher :wink: now LGTM :+1:
I will merge it as soon as all checks have passed.

**Pull request merged**